src/Troubleshooting/index.md

```diff
@@ -62,8 +64,8 @@ Vulkan development tools are required for building Vinegar with splash screen su
 - `vulkan-devel` in Arch-based distros.
 
   
-distribution that doesn't ship the development packages/tools.
-If using a distribution that doesn't ship the development packages, they may be compiled using the following guide: [Vulkan-Loader Build Instructions](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/BUILD.md#building-on-linux).
+Distributions that don't ship the development packages/tools:
+If you're using a distribution that doesn't ship the development packages, they may be compiled using the following guide: [Vulkan-Loader Build Instructions](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/BUILD.md#building-on-linux).
```

Comment:
```suggestion
> **Note:** If you're using a distribution that doesn't ship the development packages, they may be compiled using the following guide: [Vulkan-Loader Build Instructions](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/BUILD.md#building-on-linux).
```
This will show up in an infobox like on the following page: https://vinegarhq.github.io/Configuration/index.html#example-configuration